controllers/v1/programsController.js

```diff
@@ -68,13 +68,30 @@ module.exports = class Programs extends Abstract {
   async list(req) {
     return new Promise(async (resolve, reject) => {
       try {
+
+        let tenantDetails = await userService.fetchPublicTenantDetails(req.userDetails.tenantData.tenantId);
+        if (!tenantDetails.data || !tenantDetails.data.meta || tenantDetails.success !== true) {
```

Comment:
@borkarsaish65 we can optimize 
if (!tenantDetails?.success || !tenantDetails?.data?.meta) {